setup.py

```diff
@@ -49,6 +49,13 @@ def read(*rnames):
       package_dir={'': 'src'},
       install_requires=[
           'setuptools',
+
+      ],
+      setup_requires=[
+          'pytest-runner',
```

Comment:
I see, thanks for the heads up.
On the pypi page https://pypi.python.org/pypi/pytest-runner they have also a section called
```
Considerations
Conditional Requirement
```
to avoid downloading if `setup.py` is called outside of tests.  I'll play around with this at the weekend.